nano/core_test/bootstrap_server.cpp

```diff
@@ -315,7 +315,7 @@ TEST (bootstrap_server, serve_multiple)
 		{
 			// Find matching response
 			auto response_it = std::find_if (all_responses.begin (), all_responses.end (), [&] (auto ack) { return ack.id == next_id; });
-			ASSERT_TRUE (response_it != all_responses.end ());
```

Comment:
Leave it this as to was, iterators and equality asserts could cause problems. Leave as it was .